src/Benchmarks/Data/RawDb.cs

```diff
@@ -88,6 +88,45 @@ public RawDb(IRandom random, DbProviderFactory dbProviderFactory, IOptions<AppSe
             return result;
         }
 
+        public async Task<World[]> LoadMultipleUpdatesRows(int count)
+        {
+            var result = await LoadMultipleQueriesRows(count);
+
+            using (var db = _dbProviderFactory.CreateConnection())
```

Comment:
Currently this creates and closes two DB connections (one in `LoadMultipleQueriesRows`, and again in `LoadMultipleUpdatesRows`).  Perf might be better if a single DB connection was used.  Again, not sure if multiple connections was an intentional choice.  Also, connection pooling (if enabled) might mean there is little perf difference.